### PR TITLE
ZK-3938: Support screen scaling (chrome) to align column headers

### PR DESCRIPTION
**Pull request by klyvechen**

### CODE REVIEW

File: zk/src/archive/web/js/zk/dom.js [hunk 1]
```diff
@@ -800,8 +800,8 @@ zjq.prototype = {
 		if ((w = this.$()) && w._scrollbar) {// support a fake scrollbar
 			return w._scrollbar.hasVScroll();
 		}
-		var n, v;
-		return (n = this.jq[0]) && (v = n.clientWidth) && (v = n.offsetWidth - v) > 11 ? v : 0;
+		var n, v;//zk-3938: if zoom-in, the v will be smaller than 11.
+		return (n = this.jq[0]) && (v = n.clientWidth) && (v = n.offsetWidth - v) > 1 ? v : 0;
```
Comment (scribetw) [inline zk/src/archive/web/js/zk/dom.js]: 1. Why 11 or 1? Can't it be 0?
2. `hasHScroll` has a similar one, both should be modified together.

File: zk/src/archive/web/js/zk/dom.js [hunk 2]
```diff
@@ -800,8 +800,8 @@ zjq.prototype = {
 		if ((w = this.$()) && w._scrollbar) {// support a fake scrollbar
 			return w._scrollbar.hasVScroll();
 		}
-		var n, v;
-		return (n = this.jq[0]) && (v = n.clientWidth) && (v = n.offsetWidth - v) > 11 ? v : 0;
+		var n, v;//zk-3938: if zoom-in, the v will be smaller than 11.
+		return (n = this.jq[0]) && (v = n.clientWidth) && (v = n.offsetWidth - v) > 0 ? v : 0;
```
Comment (scribetw) [inline zk/src/archive/web/js/zk/dom.js]: > 2. hasHScroll has a similar one, both should be modified together.

How about `hasHScroll`?

File: zk/src/archive/web/js/zk/dom.js (hunk 1 above)
Comment (klyvechen) [reply zk/src/archive/web/js/zk/dom.js]: ok

**Comment by wenninghsu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**